dotcom-rendering/src/web/components/Callout/Form.tsx

```diff
@@ -247,7 +247,7 @@ export const Form = ({
 				{formFields.map((formField) => {
 					return (
 						<div
-							css={formFieldWrapperStyles(formField.hidden)}
+							css={formFieldWrapperStyles(!!formField.hidden)}
```

Comment:
as .hidden is now an optional field, the double bang is used here to make sure we pass through a boolean rather than undefined.